lage.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  "pipeline": {
+    "build": ["^build"],
+    "test": ["build"]
```

Comment:
Did you need to build before testing, really?

Reply:
In this repo, yeah, haha. I can fix it I guess, but that's just how y'all have this setup.